ios/Podfile.lock

```diff
@@ -1181,7 +1183,7 @@ DEPENDENCIES:
   - Analytics (~> 3.5)
   - AppAuth (~> 0.4)
   - Bolts (= 1.9.0)
-  - Branch (~> 0.27.1)
+  - Branch (~> 0.24.2)
```

Comment:
We don't want to downgrade Branch, probably your linked `@expo/xdl` is too old or sth 🤔 Try to run `et ios-generate-dynamic-macros` and then run `pod install` in `ios` folder again.